src/audio/copier.c

```diff
@@ -353,20 +354,18 @@ static struct comp_dev *copier_new(const struct comp_driver *drv,
 	list_init(&dev->bsource_list);
 	list_init(&dev->bsink_list);
 
+	/* check whether pipeline id is already taken or in use */
```

Comment:
this comment is incorrect right? we're really checking if the pipeline exists or not? Not that it has been taken or in use

Reply:
sure, copy error.